04-user-guide.Rmd

```diff
@@ -2,30 +2,20 @@
 
 This section details installation guides for
 [users](#user-installation-guide). See the [developer installation
-guide](#developer-installation-guide).
+guide](#developer-software-and-installation-guide).
 
 ## User Installation Guide
 This section describes how to install the FIMS R package and dependencies.
 
```

Comment:
I think that we should delete these lines because there is nothing in this section except line 8.